agent/src/main/java/com/microsoft/hydralab/agent/runner/xctest/XctestRunner.java

```diff
@@ -0,0 +1,99 @@
+package com.microsoft.hydralab.agent.runner.xctest;
+
+import com.android.ddmlib.IShellOutputReceiver;
+import com.microsoft.hydralab.agent.runner.TestRunner;
+import com.microsoft.hydralab.agent.runner.TestTaskRunCallback;
+import com.microsoft.hydralab.common.entity.common.DeviceInfo;
+import com.microsoft.hydralab.common.entity.common.TestRun;
+import com.microsoft.hydralab.common.entity.common.TestTask;
+import com.microsoft.hydralab.common.management.DeviceManager;
+import com.microsoft.hydralab.common.screen.ScreenRecorder;
+import com.microsoft.hydralab.common.util.Const;
+import com.microsoft.hydralab.common.util.FileUtil;
+import com.microsoft.hydralab.common.util.ShellUtils;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+
+public class XctestRunner extends TestRunner {
```

Comment:
Should this be `XctestRunner` or `XCTestRunner` according to https://developer.apple.com/documentation/xctest

Reply:
Updated